ompi/runtime/ompi_mpi_params.c

```diff
@@ -187,7 +187,7 @@ int ompi_mpi_register_params(void)
         ompi_mpi_param_check = true;
         if (!MPI_PARAM_CHECK) {
             opal_output(0, "WARNING: MCA parameter mpi_no_free_handles set to true, but MPI");
-            opal_output(0, "WARNING: parameter checking has been compiled out of Open MPI.");
+            opal_output(0, "WARNING: parameter checking has been compiled out of " OMPI_IDENT_STRING );
```

Comment:
Nit-pick: dropping the full-stop at the end of the sentence here (in case that matters to some tools)